### PR TITLE
Correction before session 2021

### PR DESCRIPTION
**Pull request by madvid**
All in the title:
* multiple typo
* change of images in module03/ex03
* clearer explanation for some exercices

### CODE REVIEW

File: module01/ex04/ex04.md [hunk 1]
```diff
@@ -1,89 +1,39 @@
...
+|                         |                                 |
+| -----------------------:| ------------------------------- |
+|   Turn-in directory:    |  ex04/                          |
+|   Files to turn in:     |  eval.py                        |
+|   Forbidden functions   |  while                          |
+|   Remarks:              |  use zip & enumerate            |
 
 ## Objective:
-The goal of the exercise is to discover the concept of generator object in Python.
+The goal of the exercise is to discover 2 useful methods for listsm tuplesm dictionnaries (iterable class
```
Comment (llenotre) [inline module01/ex04/ex04.md:11]: listsm ? tuplesm ?

File: module01/ex05/ex05.md [hunk 2]
```diff
@@ -1,37 +1,84 @@
...
+|   Turn-in directory:    |  ex05/              |
+|   Files to turn in:     |  the_bank.py        |
+|   Forbidden functions:  |  None               |
+|   Remarks:              |  n/a                |
 
 ## Objective:
-The goal of the exercise is to discover 2 useful methods for listsm tuplesm dictionnaries (iterable class
-objects more generally) named `zip` and `enumerate`.
+The goals of this excercise is to discover new built-in functions, deepen the class manipulation and
+to be aware of the posibility to modify istanced objects.
```
Comment (llenotre) [inline module01/ex05/ex05.md:12]: posibility -> possibility
istanced -> instanced

File: module01/ex05/ex05.md [hunk 3]
```diff
@@ -1,37 +1,84 @@
...
+* not corrupted,
+* and stores enough money to complete the transfer.
+
+How do we define if a bank account is corrupted? A corrupted bank account has:
+* an even number of attributes,
+* an attribute starting with `b`,
+* no attribute starting with `zip` or `addr`,
+* no attribute `name`, `id` and `value`.
+
+A transaction is invalid if `amount < 0` or if the amount is larger than the available funds of the sending account.
```
Comment (llenotre) [inline module01/ex05/ex05.md:52]: available funds -> balance ?

File: module02/ex03/ex03.md [hunk 4]
```diff
@@ -60,7 +62,7 @@ You can make the class skip lines at the top and the bottom of the file,
 and also keep the first line as a header if `header` is `True`.
 
 The file should not be corrupted (either a line with too many values or a line with too few values), otherwise return `None`.
-You have to handle the case `file not found`.
+You have also to handle the case `file not found`.
```
Comment (llenotre) [inline module02/ex03/ex03.md:65]: have also -> also have

**Review by llenotre — COMMENTED** [review]
A few typos are left

File: module01/ex04/ex04.md (hunk 1 above)
Comment (madvid) [reply module01/ex04/ex04.md:11]: not in subject for holygraph

File: module01/ex05/ex05.md (hunk 2 above)
Comment (madvid) [reply module01/ex05/ex05.md:12]: not in the subject for holygraph

File: module01/ex05/ex05.md (hunk 3 above)
Comment (madvid) [reply module01/ex05/ex05.md:52]: correction copy on the subject for the holygraph

File: module02/ex03/ex03.md (hunk 4 above)
Comment (madvid) [reply module02/ex03/ex03.md:65]: not exactly the same sentence on the subject intended for holygraph (You have to handle the case \texttt{file not found})

File: module01/ex04/ex04.md (hunk 1 above)
Comment (madvid) [reply module01/ex04/ex04.md:11]: fix

File: module01/ex05/ex05.md (hunk 3 above)
Comment (madvid) [reply module01/ex05/ex05.md:52]: fix

**Pull request closed without merging**